### PR TITLE
HybridProperty implementation with single token list

### PR DESCRIPTION
**Pull request by CheatEx**
Original review PR #230

### CODE REVIEW

File: hybrid/src/main/java/jetbrains/jetpad/hybrid/PrettyHybridProperty.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+      myInSync = true;
+      try {
+        r.run();
+      } finally {
+        myInSync = false;
+      }
+    }
+  }
+
+  private void sourceAdd(final int index, final Token item) {
```
Comment (katepol) [inline hybrid/src/main/java/jetbrains/jetpad/hybrid/PrettyHybridProperty.java]: final modifiers are redundant. See the following methods too

File: hybrid/src/main/java/jetbrains/jetpad/hybrid/PrettyHybridProperty.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    return myTokens;
+  }
+
+  @Override
+  public String getPropExpr() {
+    return "PrettyHybridProperty[" + myTokens + "]";
+  }
+
+  @Override
+  public Registration addHandler(final EventHandler<? super PropertyChangeEvent<ModelT>> handler) {
```
Comment (katepol) [inline hybrid/src/main/java/jetbrains/jetpad/hybrid/PrettyHybridProperty.java]: unnecessary final

File: hybrid/src/test/java/jetbrains/jetpad/hybrid/TestTokenWrapper.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package jetbrains.jetpad.hybrid;
```
Comment (katepol) [inline hybrid/src/test/java/jetbrains/jetpad/hybrid/TestTokenWrapper.java]: add license text, please

**Comment by katepol** [comment]
It will be OK with me when you fix my notes.

**Comment by timzam** [comment]
Looks good to me.

**Comment by azlobinjb** [comment]
@katepol done

**Pull request merged**